frontend/src/lib/components/ChartFilter/ChartFilter.tsx

```diff
@@ -58,7 +58,7 @@ export function ChartFilter({ filters, onChange, disabled }: ChartFilterProps):
     }): JSX.Element {
         return (
             <Tooltip title={tooltip} placement="left">
-                <div style={{ width: '100%' }}>
+                <div className="width-full">
```

Comment:
I think this is `w-full`

Reply:
![homer-hedge](https://user-images.githubusercontent.com/984817/183046561-2abcbc90-313e-4fc5-ac1e-597736b1d085.gif)